buildSrc/subprojects/binary-compatibility/src/main/groovy/gradlebuild/binarycompatibility/rules/AbstractGradleViolationRule.groovy

```diff
@@ -142,17 +142,43 @@ abstract class AbstractGradleViolationRule extends AbstractContextAwareViolation
             change.changes
         )
 
-        def id = "accept" + (change.type + change.member).replaceAll('[^a-zA-Z0-9]', '_')
+        def changeId = (change.type + change.member).replaceAll('[^a-zA-Z0-9]', '_')
         Violation violation = Violation.error(
             member,
-            rejection.getHumanExplanation() + """. If you did this intentionally, please accept the change and provide an explanation:
-                <a class="btn btn-info" role="button" data-toggle="collapse" href="#${id}" aria-expanded="false" aria-controls="collapseExample">Accept this change</a>
-                <div class="collapse" id="${id}">
+            rejection.getHumanExplanation() + """.
+                <br>
+                <p>
+                If you did this intentionally, please accept the change and provide an explanation:
```

Comment:
It is currently hard to identify which message belongs to which button. Do you think it helps if you move the button inside the `<p>`? Another option would be to add a `<br/>` in front of each new section. Or put the test right of the buttion, instead of adding it above it.